tensorflow/core/graph/mkl_layout_pass.cc

```diff
@@ -191,9 +195,10 @@ namespace tensorflow {
 //        B = MaxPoolGrad(X, A, Y)
 //
 //        We will transform this graph to propagate workspace as:
+//        (with contiguous ordering)
```

Comment:
Did/could you add the definition of workspace, when this term is used for the first time?